pkg/backupdriver/backup_driver_controller_base.go

```diff
@@ -403,6 +403,12 @@ func (ctrl *backupDriverController) syncSecretByKey(key string) error {
 		return err
 	}
 	ctrl.logger.Debugf("Successfully retrieved latest vSphere VC credentials.")
+	err = utils.RetrieveVddkLogLevel(params, ctrl.logger)
```

Comment:
VDDK is not used in backup-driver. So it doesn't matter whether we retrieve configured VDDK log level in backup-driver.